contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/serial_test.py

```diff
@@ -73,10 +75,14 @@ def __init__(
         self.stopbits = stopbits
         self.timeout = timeout
         self.data_size = data_size
+        self.rs485_settings = rs485_settings
         self.ser = self.serial_init(node)
         self.group = []
         for ser in group:
-            self.group.append(self.serial_init(ser))
+            try:
+                self.group.append(self.serial_init(ser))
+            except Exception:
+                raise SystemError("Failed to init RS485 port: {}".format(ser))
```

Comment:
How do we know exactly this exception is about RS485? Isn't there any possibility for other types of exceptions?

Reply:
You are right, we can't. I'll correct the phrase to more neutral.